test/sagemaker_tests/pytorch/training/integration/local/test_smppy_local.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2018-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+from __future__ import absolute_import
+
+import os
+
+import pytest
+from sagemaker.pytorch import PyTorch
+
+from ...integration import ROLE, data_dir, smppy_mnist_script
+from ...utils.local_mode_utils import assert_files_exist
+
+
+@pytest.mark.usefixtures("feature_smppy_present")
+@pytest.mark.processor("gpu")
+@pytest.mark.integration("smppy")
+@pytest.mark.model("mnist")
+@pytest.mark.skip_py2_containers
+def test_smppy_mnist_local(docker_image, sagemaker_local_session, tmpdir):
```

Comment:
Please add the fixture `@pytest.mark.skip_cpu`, similar to other GPU-DLC-only tests in the local or sagemaker test folders.